packages/esm-generic-patient-widgets-app/src/config-schema.ts

```diff
@@ -47,6 +47,18 @@ export const configSchema = {
       _default: 5,
     },
   },
+  showGraphByDefault: {
+    _type: Type.Boolean,
+    _description: 'Displayed graph by default',
+    _default: true,
+  },
+  encounterTypes: {
```

Comment:
We should have a `_description` for this property as well.

Reply:
Parameter added